ansible_collections/arista/avd/molecule/eos_designs_unit_tests_v4.0/inventory/group_vars/all.yml

```diff
@@ -1,2 +1,59 @@
 ---
 root_dir: '{{ playbook_dir }}'
+
+
+default_node_type_keys:
```

Comment:
I am not sure why this is required here.. this should be picked up from https://github.com/aristanetworks/ansible-avd/blob/6b3aacb8eac108bd1c287f2087610f530412de68/ansible_collections/arista/avd/roles/eos_designs/defaults/main/default-node-type-keys.yml#L1 or is there an issue?

Reply:
In this case for v4.0 we need the defaults to be the new data format. That is not possible, since we share the role defaults between v4.0 and older. So this is a workaround to present the vars with the new format.

Reply:
ok - thanks for the explanation